### PR TITLE
 🐛 📝 fixes/enhancements on the tooling repo-wide

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

This PR includes various fixes/enhancements on the tooling across the repository

- Improves ``watchmedo`` ignore filter (used in the ``boot.sh`` for development) to avoid unnecessary auto-restart of the apps while developing. E.g. now we can edit tests w/o triggering the full webserver to autorestart and wait a while until it restarts ( rest module takes >30secs ... !! but i leave a speed up for another PR)
- Enhances logs messages by including variable names using f-strings as ``f"{my_var=}"  -> "my_var=33"``
- 🐛 Fixes ``tail``, ``inspect``, ``shell`` recipes in ``common.service.Makefile`` 
- 🐛 Fixes ``check`` recipe in ``api/common/schemas/Makefile``
- 📝 Updated github issue template for e2e including also portal tests. (Updated generator script as well)
- Disabled ``python linting 3.10`` CI job until we have bandwidth to fix it (received multiple complaints about it)

### ``common.service.Makefile`` 
```bash
cd osparc-simcore
make up-prod

# go to any service folder
cd services/storage
make inspect
make tail
...
make shell
```
#### test include/exclude patterns

Getting the right ignore pattern sometimes is tricky because of the different implementation of the globs. I tested the ``entrypoint`` patterns using  ``watchmedo log`` as a replacement for ``watchmedo auto-restart``. E.g.
```console
$ watchmedo log \
  --recursive \
  --pattern="*.py;*/src/*" \
  --ignore-patterns="*test*;ignoreme/*" \
  --ignore-directories \
  .

on_moved(self=<watchdog.tricks.LoggerTrick object at 0x7fe547504250>, event=<FileMovedEvent: src_path='./pytest-simcore/ignoreme/ignore.py', dest_path='./pytest-simcore/ignore.py', is_directory=False>)
on_modified(self=<watchdog.tricks.LoggerTrick object at 0x7fe547504250>, event=<FileModifiedEvent: event_type=modified, src_path='./pytest-simcore/ignore.py', is_directory=False>)
on_modified(self=<watchdog.tricks.LoggerTrick object at 0x7fe547504250>, event=<FileModifiedEvent: event_type=modified, src_path='./pytest-simcore/ignore.py', is_directory=False>)
on_modified(self=<watchdog.tricks.LoggerTrick object at 0x7fe547504250>, event=<FileModifiedEvent: event_type=modified, src_path='./pytest-simcore/ignore.py', is_directory=False>)
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/e2e_report.md [hunk 1]
```diff
@@ -23,9 +22,6 @@ labels: e2e
   - [ ] [e2e/dalco/production]
   - [ ] [e2e/aws/staging]
   - [ ] [e2e/aws/production]
-  
-  
- ## known temporary fix
```
Comment (GitHK) [inline .github/ISSUE_TEMPLATE/e2e_report.md:28]: we decided to remove this?

**Review by GitHK — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/e2e_report.md (hunk 1 above)
Comment (pcrespov) [reply .github/ISSUE_TEMPLATE/e2e_report.md:28]: that was not intended ... sorry!

File: services/dask-sidecar/docker/boot.sh [hunk 2]
```diff
@@ -39,7 +39,7 @@ if [ ${DASK_START_AS_SCHEDULER+x} ]; then
   echo "$INFO" "Starting as ${SCHEDULER_VERSION}..."
   if [ "${SC_BOOT_MODE}" = "debug-ptvsd" ]; then
 
-    exec watchmedo auto-restart --recursive --pattern="*.py" -- \
+    exec watchmedo auto-restart --recursive --pattern="*.py;*/src/*" --ignore-patterns="*test*;pytest_simcore/*;setup.py;*ignore*" --ignore-directories -- \
```
Comment (sanderegg) [inline services/dask-sidecar/docker/boot.sh:42]: question, is this going to restart if I'm changing the libraries it uses? (it's a general question for all the other services as well)

File: services/dask-sidecar/docker/boot.sh [hunk 3]
```diff
@@ -71,7 +71,8 @@ else
   num_gpus=$(python -c "from simcore_service_dask_sidecar.utils import num_available_gpus; print(num_available_gpus());")
 
   # RAM (is computed similarly as the default dask-sidecar computation)
-  ram=$(($(python -c "import psutil; print(int(psutil.virtual_memory().total * $num_cpus/$(nproc)))") - ${DASK_SIDECAR_NON_USABLE_RAM:-0}))
+  _value=$(python -c "import psutil; print(int(psutil.virtual_memory().total * $num_cpus/$(nproc)))")
+  ram=$((_value - ${DASK_SIDECAR_NON_USABLE_RAM:-0}))
```
Comment (sanderegg) [inline services/dask-sidecar/docker/boot.sh:75]: a bit picky here...

**Review by sanderegg — APPROVED** [review]
👍

File: services/dask-sidecar/docker/boot.sh (hunk 2 above)
Comment (pcrespov) [reply services/dask-sidecar/docker/boot.sh:42]: yes.

this is monitoring everything below ``/devel`` folder, which binds all the packages as well.

File: services/dask-sidecar/docker/boot.sh (hunk 3 above)
Comment (pcrespov) [reply services/dask-sidecar/docker/boot.sh:75]: Sorry, the code hightligh in vscode could not render further this point and this is the only way I found to overcome this.

**Pull request merged**